poll/public/js/poll.js

```diff
@@ -10,7 +10,35 @@ function PollUtil (runtime, element, pollType) {
         this.votedUrl = runtime.handlerUrl(element, 'student_voted');
         this.submit = $('input[type=button]', element);
         this.answers = $('input[type=radio]', element);
+
+        // Set up gettext in case it isn't available in the client runtime:
+        if (typeof gettext == "undefined") {
+            window.gettext = function gettext_stub(string) { return string; };
+            window.ngettext = function ngettext_stub(strA, strB, n) { return n == 1 ? strA : strB; };
+        }
+
+        // Make gettext available in Handlebars templates
+        Handlebars.registerHelper('i18n', function(str) {
+            return gettext(str);
+        });
+
+        // Make i18n_ngettext available in Handlebars templates
```

Comment:
@asadiqbal08 I'd change this to "Make ngettext available in Handlebars templates", to be consistent with the comment for the `i18n` helper.

Same comment applies to corresponding line (L26) in poll_edit.js.